src/Elements/BaseElement.php

```diff
@@ -215,7 +215,7 @@ public function hasClass($className)
      */
     public function getTitle()
     {
-        return $this->getMetaData()['title'];
+        return $this->getMetaData()['title'] ?? '';
```

Comment:
@M165437 this would also be a reason to raise the min PHP version (related to #10)

Reply:
PHP version was raised in master now.